pkgs/development/python-modules/pydiscourse/default.nix

```
@@ -0,0 +1,34 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub ,
+  lib,
+  pythonAtLeast,
+  requests,
+}:
```

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, requests
}:
```

`pythonAtLeast` is not used.